requirements/edx/github.in

```diff
@@ -79,6 +79,8 @@
 ##############################################################################
 
 # ... add dependencies here
+# FIXME JV - move to base.in once pypi package created
+git+https://github.com/open-craft/openedx-learning.git@jill/taxonomy-api#egg=openedx-learning==0.1
```

Comment:
To do: [deploy a release for openedx-learning to pypi](https://github.com/openedx/modular-learning/issues/63#issuecomment-1598298799) and move this to base.in.